source/_integrations/wolflink.markdown

```diff
@@ -28,5 +28,10 @@ The {% term integration %} fetches all data based on parameters, that are expose
 - Wolf Link Home Pro connected to a FGB-28 device
 - Wolf ISM7e / Link Pro connected to a COB-20 device
 - Wolf ISM7 (HW v1.0, SW v2.10.47) connected to a CGB-2-14 device
+- Wolf Link Home connected to a CHA-7 IDU
+
+### Statistics with professional permissions
+
+The Smartset app can display values and statistics from the professional layer, but this integration can only retrieve user permission data.
```

Comment:
This and part of the PR description look AI generated. I'll go ahead and close this.

If I'm mistaken, then I'm sorry.

../Frenck